packages/babel-plugin-transform-unicode-property-regex/package.json

```diff
@@ -2,7 +2,7 @@
   "name": "@babel/plugin-transform-unicode-property-regex",
   "version": "7.0.0-beta.3",
   "description": "Compile Unicode property escapes in Unicode regular expressions to ES5.",
-  "homepage": "https://github.com/babel/babel/tree/master/packages/babel-plugin-transform-unicode-property-regex",
+  "homepage": "https://babeljs.io/",
```

Comment:
Other transforms in packages point to the `tree` URL. Why would this one be different?

Reply:
Looks like we handle the `homepage` key relatively inconsistently: most transforms don't have it, except transform-regenerator which has the tree URL; the rest point to the website.

Reply:
on npm

> The url to the project homepage.

So I think the site sounds right

Reply:
Hmm… The “project” in this case is this particular transform, not Babel itself (which is the parent project) IMHO. No big deal either way :)

Reply:
I'm ok with either, if we want to do the url of the package then we should change everything

Reply:
Let's decide and normalize in a followup PR?